### PR TITLE
Admin notice

### PR DESCRIPTION
**Pull request by chandonn-zz**
Related to #76

### CODE REVIEW

File: admin/partials/admin-monk-notice-render.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ *
+ * @package    Monk
+ * @subpackage Monk/Includes
+ */
+
+?>
+<div class="updated notice is-dismissible monk-banner">
+		<h2><?php esc_html_e( 'Monk has been activated, go check the settings', 'monk' ); ?></h2>
+	<p>
+		<a href="<?php esc_attr_e( 'admin.php?page=monk', 'monk' ); ?>" class="button button-primary"><?php esc_html_e( 'Go to Monk settings', 'monk' ); ?></a>
```
Comment (brenoalvs) [inline admin/partials/admin-monk-notice-render.php]: Wrong admin url and scaping, get `$settings_url` from method and `esc_url` function.

File: includes/class-monk.php [hunk 2]
```diff
@@ -164,7 +166,26 @@ private function set_locale() {
...
+	/**
+	 * Function to display a notice on plugin activation
+	 *
+	 * This function gets the user to the configuration page
+	 *
+	 * @since   1.0.0
+	 */
+	public function monk_display_activation_notice() {
+		add_option( 'monk_activated', true );
+		$settings_url = admin_url( 'admin.php?page=monk' );
```
Comment (brenoalvs) [inline includes/class-monk.php]: Align `=` signs

File: includes/class-monk.php [hunk 3]
```diff
@@ -164,7 +166,26 @@ private function set_locale() {
...
+		$monk_activated = get_option( 'monk_activated', false );
+		if ( ! empty( $monk_activated ) ) {
```
Comment (brenoalvs) [inline includes/class-monk.php]: When this option is updated?

**Review by brenoalvs — CHANGES_REQUESTED** [review]

File: admin/partials/admin-monk-notice-render.php [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+ * @link       https://github.com/brenoalvs/monk
+ * @since      1.0.0
+ *
+ * @package    Monk
+ * @subpackage Monk/Includes
+ */
+
+?>
+<div class="updated notice is-dismissible monk-banner">
+	<p><?php printf( __( '<strong>Monk:</strong> You need to <a href="%s">configure your language preferences</a>.' ), admin_url( 'admin.php?page=monk' ) ); ?></p>
```
Comment (leonofre) [inline admin/partials/admin-monk-notice-render.php]: Add text domain

**Review by leonofre — CHANGES_REQUESTED** [review]

**Review by brenoalvs — APPROVED** [review]

**Review by leonofre — APPROVED** [review]

**Pull request merged**